styles/App.scss

```diff
@@ -1,3 +1,4 @@
 @import './base/variables';
 @import './base/mixins';
 @import './global';
+@import './VisualImage';
```

Comment:
camelCase for scss components files [WIKI](https://github.com/Rednaxela700/CC-Project-2/wiki#styling)